curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-telephone-number-validator-project/build-a-telephone-number-validator.md

```diff
@@ -26,74 +26,74 @@ Note that the area code is required. Also, if the country code is provided, you
-1. When the `#user-input` element contains `(6054756961)` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: (6054756961)`
-1. When the `#user-input` element contains `2 (757) 622-7382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 2 (757) 622-7382`
-1. When the `#user-input` element contains `0 (757) 622-7382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 0 (757) 622-7382`
-1. When the `#user-input` element contains `-1 (757) 622-7382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: -1 (757) 622-7382`
-1. When the `#user-input` element contains `2 757 622-7382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 2 757 622-7382`
-1. When the `#user-input` element contains `10 (757) 622-7382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 10 (757) 622-7382`
-1. When the `#user-input` element contains `27576227382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 27576227382`
-1. When the `#user-input` element contains `(275)76227382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: (275)76227382`
-1. When the `#user-input` element contains `2(757)6227382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 2(757)6227382`
-1. When the `#user-input` element contains `2(757)622-7382` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 2(757)622-7382`
-1. When the `#user-input` element contains `555)-555-5555` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 555)-555-5555`
-1. When the `#user-input` element contains `(555-555-5555` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: (555-555-5555`
-1. When `#user-input` contains `(555)5(55?)-5555` and `#check-btn` is clicked, `#results-div` should contain the text `Invalid US number: (555)5(55?)-5555`
-1. When the `#user-input` element contains `55 55-55-555-5` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 55 55-55-555-5`
-1. When the `#user-input` element contains `11 555-555-5555` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `Invalid US number: 11 555-555-5555`
+1. When the `#user-input` element contains `1 555-555-5555` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `"Valid US number: 1 555-555-5555"`
```

Comment:
```suggestion
1. When the `#user-input` element contains `"1 555-555-5555"` and the `#check-btn` element is clicked, the `#results-div` element should contain the text `"Valid US number: 1 555-555-5555"`
```

the phone numbers are strings too

Reply:
that is correct, the number in this case would be a string, but thats is the result of it being input by a user in ` <input maxlength="20" type="text" id="user-input" value=""> ` my understanding of the issue was relating the campers confusion of ` Valid US number: 1 555-555-5555 ` and ` "Valid US number: 1 555-555-5555" ` as requirement for the tests.